docs/how_to/use_optimizationlevel.rst

```diff
@@ -0,0 +1,278 @@
+#####################################
+Setting transpiler optimization level
+#####################################
+
+This guide shows you how to use the ``optimization_level``
+parameter with :func:`~qiskit.compiler.transpile`.
```

Comment:
The `~` results in only showing `transpile` rather than `qiskit.compiler.transpile`. Imo it is useful to show the whole import path.

```suggestion
parameter with :func:`qiskit.compiler.transpile`.
```

Reply:
that is ok because just put a link in the method, and the oder tutorials use the same format